### ISSUE TITLE
Idea: say that you are in alpha

### ISSUE THREAD

**Issue by Hypher** [OpenShot/openshot-qt#2396]
**Is your feature request related to a problem? Please describe.**
Seriously.... a problem? **A** problem? My god...... sooo many bugs, but, the worst of all, is that the exported video DOES FUCKING NOT LOOK LIKE THE PREVIEW IN YOUR SOFTWARE

**Describe the solution you'd like**
Please put a warning on the installer and at first start of your software that says: "Alpha version still in development, please don't use for anything productive."

**Describe alternatives you've considered**
Hacking your server / killing all of you / .... nothing legal though

**Additional context**
I'm serious, put that fucking warning.
You owe me and probably to a lot of people many wasted hours. I hate you.

**Comment by Hypher** [comment]
Yeah, I was mad after 7 hours of video editing, seeing that I could not get anything from my hard work.
For your information the two major problems I faced were:
- I configured the project for 1080i 25fps, but exporting in 1080i 25fps gave me an all-buggy rendering where many of my keyframes were messed up, elements showing not in time or with wtf-values, and no fadeout (decreasing alpha). I figured out after a while that changing the framerate of the export (eg 30fps) fixed that mess.
- Raising audio clip volume above 1 is ok inside the editor, but saturates harshly on high-frequencies in the exported video.
Gloabally, seems not to be a WYSIWYG video editor, which is... slightly annoying for such tool.
Just about sound, #1598, #1656 (both closed for inactivity and probably not fixed) or #2354 (still open) illustrates the problems your users are facing.
Closing issues for inactivity is a good way to have only 581 open issues, but will not make bugs disappear.
I understand that having more than 2 or 3 serious open-source video editing projects in the world does not help to get a sufficient team that can handle the complexity of such development...
But yeah, what I'm mainly adressing (under the real anger I got when I wrote my report), is to warn your users about. Because there is nothing more frustrating than spending hours for nothing.

**Comment by peanutbutterandcrackers** [comment]
> undo log does not work (first step does, but rarely more than that)

@ferdnyc - I think here is another scarlet thread for you, good sir.

@Hypher - Yes, I understand. I myself have felt similarly. However, many devs will soon be back from their prolonged absence, and we might be able to see some progress soon.
As for me, because I know my way around, and can use some other Linux Command Line Utilities, I am fairly confident with my abilities to make OpenShot get up and do the job for me. However, I understand that normal users might not be as knowledgeable. That is why I still love OpenShot and am volunteering for it to get better. And hopefully, soon, we'll have a great Free/Libre Video Editor for everyone.
Thank you for your input. I will try to get the devs' focus on this matter. Might be a while, but I'll try.

**Comment by ferdnyc** [comment]
@peanutbutterandcrackers 
> > undo log does not work (first step does, but rarely more than that)
> 
> @ferdnyc - I think here is another scarlet thread for you, good sir.

I don't know what that means, nor what this has to do with me.

**Comment by ferdnyc** [comment]
@peanutbutterandcrackers : There is no such investigation. History should be removed from the `.osp` files. But that's unrelated to the issue of the undo _functionality_, anyway. 

I haven't experienced undo not working beyond the first history step, nor have I seen any other reports of such issues... have you?

**Comment by peanutbutterandcrackers** [comment]
@ferdnyc - Undo does work for me, but I do have to wait for a while. Sometimes, it does seem like it isn't, in which case I try to undo again. Perhaps undo needs a little more redoing? *ba dum tss*
